beater/config.go

```diff
@@ -14,6 +14,7 @@ type Config struct {
 	SecretToken        string        `config:"secret_token"`
 	SSL                *SSLConfig    `config:"ssl"`
 	ConcurrentRequests int           `config:"concurrent_requests" validate:"min=1"`
+	EnableFrontend     bool          `config:"enable_frontend"`
```

Comment:
could you add the setting to the config_tests?

Reply:
agh sorry, i was too fast :(
didn't know about config_tests, will do in separate PR